proto/p4/v1/p4runtime.proto

```diff
@@ -447,6 +447,13 @@ message PacketReplicationEngineEntry {
   }
 }
 
+// A backup replica used as a fallback port for a replica.
+message BackupReplica {
+  // Added in v1.5.0.
+	bytes port = 1;
+	uint32 instance = 2;
```

Comment:
Not a big deal, but it looks like there is some tabs vs. whitespace weirdness going on in the definition of BackupReplicate.